modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -184,17 +184,21 @@ public void archiveOrganization(TenantInfo tenantInfo) throws RuntimeProvisionin
             kubClient = KubernetesProvisioningUtils.getFabric8KubernetesClient();
             DeploymentList deploymentList = kubClient.extensions().deployments().list();
 
-            if (deploymentList.getItems().contains(deployment)) {
+            Deployment currentDeployement = kubClient.inNamespace(namespace.getMetadata().getName()).extensions()
```

Comment:
for readability of the line shouldnt we consider seperating the line into several parts?